src/index.html

```diff
@@ -16,16 +17,19 @@
       href="style.css"
     />
   </head>
+
   <body>
     <input
+      class="search-bars search-bars-big"
       type="text"
-      data-qa="keypress"
+      data-qa="big"
       placeholder="Try “Los Angeles“"
     />
 
     <input
+      class="search-bars search-bars-small"
       type="text"
-      data-qa="hover"
+      data-qa="small"
       placeholder="Try “Los Angeles“"
     />
```

Comment:
The input fields are missing the 'name' attribute. This attribute is important as it is used when sending data in a form submission. Please, add the 'name' attribute to your input fields.